### PR TITLE
Update to swarm v4.3.0 and remove deprecations

### PR DESCRIPTION
**Pull request by gavin-norman-sociomantic**

### CODE REVIEW

**Comment by gavin-norman-sociomantic** [comment]
Damn, there's a bug in swarm v4.3.0 :(

I even tested this PR locally before releasing v4.3.0, but I must have done something wrong.

**Comment by gavin-norman-sociomantic** [comment]
Updated to use swarm v4.3.1.

File: src/dhtproto/client/internal/NodeHashRanges.d [hunk 1]
```diff
@@ -85,15 +86,15 @@ public final class NodeHashRanges : NodeHashRangesBase
 
     ***************************************************************************/
 
-    public void getFromNode ( hash_t h, ref NodeHashRange[] node_hash_ranges,
-        UseNodeDg use_node,
+    public void getFromNode ( hash_t h,
+        VoidBufferAsArrayOf!(NodeHashRange) node_hash_ranges, UseNodeDg use_node,
```
Comment (nemanja-boric-sociomantic) [inline src/dhtproto/client/internal/NodeHashRanges.d:90]: Nice that this doesn't leak anywhere to the user :+1:

File: src/fakedht/DhtNode.d [hunk 2]
```diff
@@ -91,7 +91,7 @@ public class DhtNode
         params.node_info = this;
 
         Options neo_options;
-        neo_options.cmd_handlers = request_handlers;
+        neo_options.requests = requests;
```
Comment (nemanja-boric-sociomantic) [inline src/fakedht/DhtNode.d:94]: Migration notes?

Comment (nemanja-boric-sociomantic) [reply src/fakedht/DhtNode.d:94]: Or is this just a deprecation fix?

Comment (gavin-norman-sociomantic) [reply src/fakedht/DhtNode.d:94]: Yeah, this is internal stuff in the node, fixing a deprecation in swarm. Nothing related to the user.

File: src/dhtproto/client/internal/NodeHashRanges.d (hunk 1 above)
Comment (gavin-norman-sociomantic) [reply src/dhtproto/client/internal/NodeHashRanges.d:90]: Yeah, the user has no idea about the node hash range registry. Purely internal client stuff.

File: src/fakedht/DhtNode.d (hunk 2 above)
Comment (nemanja-boric-sociomantic) [reply src/fakedht/DhtNode.d:94]: Yeah, thanks for the clarification.

**Review by nemanja-boric-sociomantic — APPROVED** [review]

**Pull request merged**